robottelo/performance/perf_candlepin.py

```diff
@@ -0,0 +1,116 @@
+"""Test utilities for writing Candlepin tests
+
+Part of functionalities of Candlepin are defined in this module
+and have utilities of single register by activation-key, single
+register and attach, single subscription deletion.
+
+"""
+import logging
+import time
+import requests
+
+from robottelo.common import conf, ssh
+from robottelo.common.helpers import get_server_credentials, get_server_url
+from urlparse import urljoin
+
+
+class CandlepinPerfTest(object):
+    """Measures performance of RH Satellite 6
+
+    Candlepin Subscription functionality
+
+    """
+
+    logger = logging.getLogger('robottelo')
+
+    # parameters for Candlepin Authentication
+    username = conf.properties['foreman.admin.username']
+    password = conf.properties['foreman.admin.password']
+    serverhost = conf.properties['main.server.hostname']
+
+    requests.packages.urllib3.disable_warnings()
```

Comment:
Please move this to the line below https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/__init__.py#L39. Make sure to import the requests module on that module.